res/photon/index.html

```diff
@@ -225,6 +225,15 @@ <h3 class='photon-title-20'>Photon Message Bar Generic</h3>
             aria-label="Hide the message" title="Hide the message">
           </button>
         </div>
+        <div class="photon-message-bar multi-line">
+        This is a dismissable message bar.
+       <button class="photon-button photon-message-bar-close-button" type="button"
+          aria-label="Hide the message" title="Hide the message">
+        </button>
+        <button class="photon-button photon-button-micro photon-message-bar-action-button" type="button">
+          Call to action
+        </button>
+    </div>
```

Comment:
Btw, the indentation of the divs and buttons don't look correct to me. Could you fix it?